CreateSnapshot/src/main/java/org/opensearch/migrations/CreateSnapshot.java

```diff
@@ -59,6 +61,10 @@ public static class Args {
             "--s3-role-arn" }, required = false, description = "The role ARN the cluster will assume to write a snapshot to S3")
         public String s3RoleArn;
 
+        @Parameter(names = {
+            "--index-allowlist" }, required = false, description = "A comma separated list of indices to include in the snapshot. If not provided, all indices are included.")
```

Comment:
Do we want to align this with document migration which excludes system indicies?

Reply:
I lean towards including them by default. I'm fine with not transferring them by default, but if a user ends up wanting to move some/all of them, it would be very annoying to discover that the snapshot didn't include them.